isis/src/base/objs/LineScanCameraGroundMap/LineScanCameraGroundMap.cpp

```diff
@@ -470,22 +470,22 @@ namespace Isis {
         root.removeAt(i);
       }
     }
-     
+
     // return the calculated roots to the original system
     for (int i=0; i<root.size(); i++) {
       root[i] = root[i]/scale + timeAverage;
     }
 
     // THIS IS A PREMATURE FAILURE RETURN. IT SHOULD TRY THE NEXT METHON BEFORE FAILING
-    if (root.size() == 0) {
-      return Failure;  // there are apparently not any roots on this image
-    }
+    // if (root.size() == 0) {
```

Comment:
This was causing setUniversalGround to fail. Since there was an existing comment that this was a premature failure, I went ahead and commented this out.

Reply:
:+1: I'm not sure if leaving this commented out or trying to find the "real" issue is better (if there is one)

Reply:
I'm a little concerned about this since it's not specific to EIS, it seems like other cameras / models could be affected by this change.  Do we know what's causing the errror?

Reply:
I agree. This function is very messy. Seems like there was a method tacked on that is suppose to be tried before the function fails altogether. This if statement is causing the function to fail before it tries the next method. When I commented this out, the function returned successful using the final method. Since there was a comment added saying that this if statement was causing premature failures, it seems like no one has really addressed this issue and it probably should be looked into.

Reply:
I reimplemented this function to include the if statement but to return failure after all methods are tried. I was incorrect in stating that the function was returning successful in another method. Instead, it was incorrectly returning successful in the method in which I commented out this if statement. (This is now corrected).  This may be the reason my SetUniversalGround was causing issues with the incorrect outputted Line. Now my tests are failing SetUniversalGround due to this particular function now returning Failure.